cmd/synthetic-monitoring-agent/main.go

```diff
@@ -74,15 +75,17 @@ func run(args []string, stdout io.Writer) error {
 			AutoMemLimit         bool
 			MemLimitRatio        float64
 			DisableK6            bool
+			DisableUsageReports  bool
 		}{
-			GrpcApiServerAddr: "localhost:4031",
-			HttpListenAddr:    "localhost:4050",
-			K6URI:             "sm-k6",
-			K6BlacklistedIP:   "10.0.0.0/8",
-			SelectedPublisher: pusherV2.Name,
-			TelemetryTimeSpan: defTelemetryTimeSpan,
-			AutoMemLimit:      true,
-			MemLimitRatio:     0.9,
+			GrpcApiServerAddr:   "localhost:4031",
+			HttpListenAddr:      "localhost:4050",
+			K6URI:               "sm-k6",
+			K6BlacklistedIP:     "10.0.0.0/8",
+			SelectedPublisher:   pusherV2.Name,
+			TelemetryTimeSpan:   defTelemetryTimeSpan,
+			AutoMemLimit:        true,
+			MemLimitRatio:       0.9,
+			DisableUsageReports: true,
```

Comment:
Mostly formatting changes here. I've added `DisableUsageReports` since bool's zero value is false, and this ensures we don't start collecting usage reports before updating documentation.